<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newArticleWorkspaceWindow</class>
 <widget class="QDialog" name="newArticleWorkspaceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Article Workspace</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Article Title:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Writer:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="articleTitleTextField">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>471</width>
     <height>21</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Add Article Title...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Section:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="descriptionTextField">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Photographer:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sectionComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="writerComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="photographerComboBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Article File:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>121</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose File...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="articleFileTextField">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>111</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>390</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Issue Date:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="issueDateEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>390</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
